core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -47,18 +61,48 @@ func NewLogPollerWrapper(routerContractAddress common.Address, pluginConfig conf
 	}
 	blockOffset := int64(pluginConfig.MinIncomingConfirmations) - 1
 	if blockOffset < 0 {
+		lggr.Warnw("invalid minIncomingConfirmations, using 1 instead", "minIncomingConfirmations", pluginConfig.MinIncomingConfirmations)
 		blockOffset = 0
 	}
+	requestBlockOffset := int64(pluginConfig.MinRequestConfirmations) - 1
+	if requestBlockOffset < 0 {
```

Comment:
To my non-Go brain these checks seem like a weird way to validate a configuration file.
Basically this is enforcing that `pluginConfig.MinRequestConfirmations != 0`

It seems like if something is misconfigured this would be hard to catch, you would have to grep for a very certain log.

@bolekk is there no way to enforce bounds in another place?

Reply:
This is a good place to validate (constructor method) and we could return an error instead of re-setting to a correct value. In this case I'm in favor of resetting to 0, as it is the only invalid value (uint type). If somebody mistakenly sets zero, they likely meant 1 anyway.